tests/conftest.py

```diff
@@ -126,3 +126,84 @@ def x12_270_control_footer() -> str:
             "IEA*1*000000907~",
         ]
     )
+
+
+@pytest.fixture
+def x12_270_subscriber_transaction() -> str:
```

Comment:
@ccorley - I've cleaned up the test cases a bit. I think this will be a better structure moving forward. At a high level we will place our transaction fixtures in the conftest and roll a separate module for each transaction type rather than nesting things within a package.

Additionally, I am including a minimal number of test cases as the transactions will include ALL the segments that could be sent for each use-case, just to verify parsing and transforms.